workflow/artifacts/http/util.go

```diff
@@ -0,0 +1,41 @@
+package http
```

Comment:
maybe rename this file `client_cert.go`

Reply:
i would go for just `clients.go`, since the oauth2 client is also in there